src/main/java/Task.java

```diff
@@ -0,0 +1,55 @@
+public class Task {
+    protected String taskName;
+    protected boolean isCompleted;
```

Comment:
I think that it is good that the boolean follows the coding standard and is named to sound like a boolean with the 'is' prefix